source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/calculation/CalculationSetOptions.java

```diff
@@ -38,7 +38,7 @@ public class CalculationSetOptions implements Serializable {
   private boolean useInCombinationArchive;
   private final ArrayList<Substance> substances = new ArrayList<>();
   private final Set<EmissionResultKey> emissionResultKeys = new HashSet<>();
-  private WNBCalculationOptions wnbCalculationOptions = new WNBCalculationOptions();
+  private OwN2000CalculationOptions own2000CalculationOptions = new WNBCalculationOptions();
```

Comment:
```suggestion
  private OwN2000CalculationOptions own2000CalculationOptions = new OwN2000CalculationOptions();
```

Reply:
Think it's required for the cast in `getWnbCalculationOptions`. Will probably require a cleanup PR to change this once other repository is up to date.

Reply:
Yes that is required because otherwise the old `getWnbCalculationOptions` won't work. When the pr in Calculator is merged I'll do a followup pr to clean up the deprecated code here. It might be an idea to do this when we change the IMAER-java version pom version, so it's a bit more separated.